packages/dropdown/src/tokens.scss

```diff
@@ -0,0 +1,4 @@
+$option-height: 36px;
+
+$menu-max-height: $option-height * 10;
+$menu-top-offset: 43px; // Height from the top of the TextField wrapper to the input bottom border. @TODO: can be calculated dynamically?
```

Comment:
This number magically positions the dropdown menu right below the text input border. I think it'll require saving `refs` to the TextField for this value to be calculated dynamically. I started including these refs in https://github.com/Autodesk/hig/pull/983.